### DISCUSSION TITLE
How to add dicts to Linux's AppImage or Linux deb?

### DISCUSSION THREAD

**Discussion by zakimohzani** [koreader/koreader#7504] [Q&A] [answered]
The title says it all.

**Comment by Frenzie** [comment, marked as answer]
That would be in $HOME/.config/koreader/data/dict/ or possibly also $HOME/.stardict/dic/ (don't recall otoh).

**Comment by zakimohzani** [comment]
Thanks for replying. Will it also work for AppImage?
________________________________
Von: Frans de Jonge ***@***.***>
Gesendet: Sonntag, 4. April 2021 18:05
An: koreader/koreader
Cc: zakimohzani; Author
Betreff: Re: [koreader/koreader] How to add dicts to Linux's AppImage or Linux deb? (#7504)


That would be in $HOME/.config/koreader/data/dict/ or possibly also $HOME/.stardict/dic/ (don't recall otoh).

**Reply by Frenzie** [reply]
Yes.

**Reply by zakimohzani** [reply]
Phew.. work got in the way. Thanks for the info.

It is $HOME/.config/koreader/data/dict/ by the way.
